gateway/protocol.go

```diff
@@ -50,6 +51,11 @@ type Protocol interface {
 	// 	- Returns the list of all service IDs for which the gateway is configured to serve.
 	ConfiguredServiceIDs() map[protocol.ServiceID]struct{}
 
+	// GetSanctionedEndpoints returns the sanctioned endpoints for a given service ID.
+	// It is called by the router to allow quick information about currently sanctioned endpoints.
```

Comment:
Make the comment more "product user/dev" focused.

E.g. it is called by the router to expose the data via an API.